### PR TITLE
Nfdiv-1157 CYA page for joint CO

### PR DESCRIPTION
**Pull request by lara-mpt**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-1157

### Change description ###
 CYA page for joint CO

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/index.ts [hunk 1]
```diff
@@ -52,6 +52,8 @@ const getNextIncompleteStep = (
 
       return nextStep
         ? getNextIncompleteStep(data, nextStep, sequence, removeExcluded, checkedSteps.concat(step))
+        : [State.ConditionalOrderDrafted, State.ConditionalOrderPending].includes(data.state)
```
Comment (velimd) [inline src/main/steps/index.ts:55]: i think this needs to be its own if/case statement.

File: src/main/steps/applicant1/check-your-conditional-order-answers/template.njk [hunk 2]
```diff
@@ -6,6 +6,7 @@
 {% set nextIncompleteStep = getNextIncompleteStepUrl() %}
 {% set isConditionalOrderAnswersComplete =
         '/check-your-conditional-order-answers' in nextIncompleteStep or
+        nextIncompleteStep === '/applicant2/' or
```
Comment (velimd) [inline src/main/steps/applicant1/check-your-conditional-order-answers/template.njk:9]: is there a page which is just '/applicant2/'?

File: src/main/steps/index.ts (hunk 1 above)
Comment (lara-mpt) [reply src/main/steps/index.ts:55]: as in split that return statement into multiple if/else or that the check for that should be separate from the nextStep check? 

I thought it was worth keeping it in the nextStep check to make sure the continue-with-your-application question had been answered

File: src/main/steps/applicant1/check-your-conditional-order-answers/template.njk (hunk 2 above)
Comment (lara-mpt) [reply src/main/steps/applicant1/check-your-conditional-order-answers/template.njk:9]: There isn't but I double checked and this is needed for app2 as the getNextIncompleteStepUrl returns '/applicant2' for them when they finish the sequence

File: src/main/steps/index.ts (hunk 1 above)
Comment (velimd) [reply src/main/steps/index.ts:55]: just make it 1 if/else statement as i feel like the inline condition should really be used for if or else but when theres multiple conditions it should just be its own if (nextstep) else if (this state) else ....

File: src/main/steps/applicant1/check-your-conditional-order-answers/content.ts [hunk 3]
```diff
@@ -16,12 +16,20 @@ const en = ({ isJointApplication, isDivorce, userCase }: CommonContent) => ({
     isInformationCorrect: 'Is the information in this application still correct?',
   },
   stepAnswers: {
-    continueApplication: userCase.applicant1ApplyForConditionalOrder,
-    isInformationCorrect: userCase.applicant1ConfirmInformationStillCorrect,
+    continueApplication: `${
```
Comment (velimd) [inline src/main/steps/applicant1/check-your-conditional-order-answers/content.ts:19]: how comes here its continueApplication and actually property names and in check your answers its line1, line2...?

**Review by velimd — APPROVED** [review]

File: src/main/steps/applicant1/check-your-conditional-order-answers/content.ts (hunk 3 above)
Comment (lara-mpt) [reply src/main/steps/applicant1/check-your-conditional-order-answers/content.ts:19]: @velimd it was previously called continueApplication and isInformationCorrect, I can update it to line1 etc to be more consistent if you'd prefer?

Comment (velimd) [reply src/main/steps/applicant1/check-your-conditional-order-answers/content.ts:19]: na its kl, was just curious hence i still approved.

**Pull request merged**